content/ADR-116-fast-bootstrapping.md

```diff
@@ -98,3 +101,4 @@ In addition, there will be performance improvements in the snapshots generation
 # Key Dependencies and Open Questions
 
 - How will the transition be where multiple Catalysts have a different behavior in `/snapshots`?
```

Comment:
This doesn't look so much like open question (and definitely not a dependency), it seems more like an FAQ kind of question. I would phrase it as a statement (rather than question / answer) and place it at the end of the Approach section.

Reply:
Changed to FAQ section